stock_available/models/res_config_settings.py

```diff
@@ -28,15 +28,6 @@ def _get_stock_available_mrp_based_on(self):
         "This installs the module stock_available_immediately.",
     )
 
-    module_stock_available_sale = fields.Boolean(
```

Comment:
@marielejeune Why removing this one in particular ?

Reply:
This boolean was removed from res_config_settings.xml in an old migration (11->12 I think). 
Moreover  the module seems to be unused since v12.